src/Core/src/Platform/iOS/MauiView.cs

```diff
@@ -657,5 +657,36 @@ public override void MovedToWindow()
 
 			UpdateKeyboardSubscription();
 		}
+
+		public override bool CanBecomeFirstResponder => true;
```

Comment:
Don't set these ourselves 

The main purpose of this pr is if someone enables these then we will properly propagate focused from the Mauiview to the xplat code

Reply:
@PureWeen , I have updated the test case. Please let me know if you have any concerns.